app/src/main/res/layout/fragment_new_profile.xml

```diff
@@ -24,115 +24,122 @@
                 android:layout_height="match_parent"
                 android:orientation="vertical">
 
-            <LinearLayout
-                    android:id="@+id/editTextParentLinearLayout"
+            <ScrollView
                     android:layout_width="match_parent"
                     android:layout_height="wrap_content"
                     android:layout_marginTop="100dp"
-                    android:gravity="center"
-                    android:orientation="vertical">
-
-
-                <com.google.android.material.textfield.TextInputLayout
-                        android:id="@+id/toDoCustomTextInput"
-                        style="@style/Widget.MaterialComponents.TextInputLayout.OutlinedBox"
-                        android:layout_width="match_parent"
-                        android:layout_height="wrap_content"
-                        android:layout_gravity="center"
-                        android:layout_marginLeft="32dp"
-                        android:layout_marginRight="32dp"
-                        android:hint="@string/profile_name">
-
-                    <com.google.android.material.textfield.TextInputEditText
-                            android:id="@+id/userToDoEditText"
-                            android:layout_width="match_parent"
-                            android:layout_height="wrap_content"
-                            android:inputType="textAutoCorrect|textCapSentences"
-                            android:lines="2"
-                            android:maxLines="3"
-                            android:textColor="@color/textColor"
-                            android:textColorHint="@color/textColor" />
-
-                </com.google.android.material.textfield.TextInputLayout>
+                    android:paddingBottom="25dp">
```

Comment:
Why add this bottom margin?

Reply:
Sorry, I was experimenting something. Forgot to remove it. Correcting and pushing the code.

Reply:
<img src="https://user-images.githubusercontent.com/61552810/95558175-76f04600-0a33-11eb-801d-c4b7735c4434.png" width="450px" height="225px">
<img src="https://user-images.githubusercontent.com/61552810/95558183-7952a000-0a33-11eb-9f65-2a102b32b587.png" width="450px" height="225px">
<img src="https://user-images.githubusercontent.com/61552810/95558196-7d7ebd80-0a33-11eb-9908-a20dac575482.png" width="250px" height="480px">

Reply:
I don't think the position of the submit button is as expected.

Reply:
yes, I had to shift it little up, otherwise the gap was very much in landscape mode.